tests/unit/dojson/test_dojson_hep_bd9xx.py

```diff
@@ -403,6 +403,8 @@ def test_references_from_999C5a_h_o_s_x_y_0():
     snippet = (
         '<datafield tag="999" ind1="C" ind2="5">'
         '  <subfield code="a">doi:10.1142/S0217751X0804055X</subfield>'
+        '  <subfield code="a">hdl:10138/168995</subfield>'
```

Comment:
please don't modify existing snippets, they represent pieces of existing records (as referenced in the comment following the snippet). If you need another example, please add a new test case with a real record snippet.